cmvs_catkin/thirdparty/graclus1.2/multilevelLib/mlkkm.c

```diff
@@ -17,7 +17,7 @@
 //#include <driver.h>
 //#include "../spectralLib/driver.h"
 
-extern int spectral_initialization;
+extern int spectral_initialization __attribute__((weak_import));
```

Comment:
Probably not, but it allows the compiler to do more error checking so it surely doesn't hurt. I'm gonna leave it here.